client/src/api/addEvent.js

```diff
@@ -0,0 +1,10 @@
+export function setEvent(title, date, descr, background, user) {
+  const datas = { title, date, descr, background, owner: user };
```

Comment:
I don't think `datas` is a correct word :).
Instead, why not call it `event`?

And it could make sense to only have one argument here like `setEvent(event) `